lxd/project/project.go

```diff
@@ -11,8 +11,8 @@ const Default = "default"
 // separator is used to delimit the project name from the suffix.
 const separator = "_"
 
-// Instance Adds the "<project>_" prefix to instance name when the given project name is not "default".
-func Instance(projectName string, instanceName string) string {
+// Instance adds the "<project>_" prefix to instance name when the given project name is not "default".
+func Instance(projectName, instanceName string) string {
```

Comment:
I personally avoid this syntax as it makes finding the type somewhat confusing.
My brain is trained to look for `<name> <type>` so I'd be wondering what `instanceName` is for a type rather than immediately notice that it's the next variable name.

Reply:
@stgraber I see your point, will fix.